packages/server/src/faker.ts

```diff
@@ -0,0 +1,14 @@
+// @ts-ignore
+import Faker from 'faker/lib';
+
+export function createFaker(seed: number): any {
+  let faker = new Faker({ locales: require('faker/lib/locales') });
```

Comment:
why do we need to use require here?

Reply:
We _may_ want to defer to loading locales depending on the simulation, but best to keep it simple for now. I'm just going to import it.